packages/plugins/plugin-clevertap/src/ClevertapPlugin.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  DestinationPlugin,
+  PluginType,
+  TrackEventType,
+  ScreenEventType,
+  IdentifyEventType,
+  generateMapTransform,
+} from '@segment/analytics-react-native';
+
+import { mapTraits, transformMap } from './parameterMapping';
+import CleverTap from 'clevertap-react-native';
+const mappedTraits = generateMapTransform(mapTraits, transformMap);
+
+export class ClevertapPlugin extends DestinationPlugin {
+  type = PluginType.destination;
+  key = 'CleverTap';
+
+  identify(event: IdentifyEventType) {
+    const traits = event.traits as Record<string, unknown>;
+    const safeTraits = mappedTraits(traits);
+    const userId = event.userId ?? event.anonymousId;
+
+    if (event.traits?.birthday !== undefined) {
```

Comment:
Maybe try switching to using `safeTraits.DOB` everywhere since it's kinda confusing that it is reading the event but deleting the property in the `DOB`